manifests/j/JFrog/Conan/2.12.2/JFrog.Conan.installer.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: JFrog.Conan
+PackageVersion: 2.12.2
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /ALLUSERS
+UpgradeBehavior: install
+ProductCode: Conan Package Manager_is1
+ReleaseDate: 2025-02-12
+AppsAndFeaturesEntries:
+- ProductCode: Conan Package Manager_is1
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '{code:DefDirRoot}\Conan'

```

Comment:
This doesn't look correct

Reply:
@russellbanks - It seems like this might be an issue with Komac??

Looking at the version history, `InstallationMetadata` was removed from `JFrog.Conan` version `2.11.0` but was re-introduced by a PR made with Komac for `2.12.1` - https://github.com/microsoft/winget-pkgs/pull/217916

Reply:
Thanks for the ping - this is taken from the Inno install directory and usually parts in `{}` are converted to relative paths (e.g. `{pf64}` -> `%ProgramFiles%`) but as this references compiled code, it can't be converted. In this case, I've changed it to ignore it - https://github.com/russellbanks/Komac/commit/727cce42e3c1d9055379f265bd0e26c8fb61d05b.

Reply:
Thanks! I think I've seen the same behavior a few other places like `ProductCode`, but it may take me a bit to find an example

Reply:
@exoosh no need to resubmit with komac built from the repo (I'll release a new version soon anyway). I would just remove the `InstallationMetadata` (which includes the `DefaultInstallLocation` sub-field) field for now.